display/Aurebesh10Font.h

```diff
@@ -0,0 +1,1507 @@
+const uint8_t Aurebesh10pt7bChar0[] = {
```

Comment:
What is the licenses of the aurabesh font?
Are we allowed to use it in ProffieOS?
Is the license compatible with GPL?

Reply:
The one I converted is by BobaFonts, actually named Aurek-Besh. 
https://www.fontspace.com/boba-fonts
It looks like it maybe the same source as StarJedi?
Licencing is Personal Use, defined there as "typically refers to any use that is not meant to generate profit."
Something specifically labeled GPL would be this version:
https://fonts2u.com/aurebesh-af-canon.font
It seems there's a lot out there, so considering it's re-formatted using fontconvert as well, I don't think there's an issue? I'd be happy to investigate further if need be.

Reply:
Where did you see the "personal use" license? Because that could be a problem.
From what I can see, both StarJedi and Aurek-Besh were made by the same person, and they seem to be licensed as freeware with no restrictions., which would mean that everything is fine.

Reply:
Oh. It was on the FAQ page of that site, https://www.fontspace.com/help#license-3

Reply:
But the font's page specifically says Licenced as Freeware

Reply:
I think the license is fine then.